tensorflow-core/tensorflow-core-examples/src/main/java/org/tensorflow/examples/LabelImage.java

```diff
@@ -0,0 +1,235 @@
+      System.err.println("Failed to read [" + path + "]: " + e.getMessage());
+      System.exit(1);
+    }
+    return null;
+  }
+
+  private static List<String> readAllLinesOrExit(Path path) {
+    try {
+      return Files.readAllLines(path, Charset.forName("UTF-8"));
+    } catch (IOException e) {
+      System.err.println("Failed to read [" + path + "]: " + e.getMessage());
+      System.exit(0);
+    }
+    return null;
+  }
+
+  // In the fullness of time, equivalents of the methods of this class should be auto-generated from
+  // the OpDefs linked into libtensorflow_jni.so. That would match what is done in other languages
+  // like Python, C++ and Go.
+  static class GraphBuilder {
+    GraphBuilder(Graph g) {
+      this.g = g;
+    }
+
+    Output<Float> div(Output<Float> x, Output<Float> y) {
+      return binaryOp("Div", x, y);
+    }
+
+    <T> Output<T> sub(Output<T> x, Output<T> y) {
+      return binaryOp("Sub", x, y);
+    }
+
+    <T> Output<Float> resizeBilinear(Output<T> images, Output<Integer> size) {
+      return binaryOp3("ResizeBilinear", images, size);
+    }
+
+    <T> Output<T> expandDims(Output<T> input, Output<Integer> dim) {
+      return binaryOp3("ExpandDims", input, dim);
+    }
+
+    <T, U> Output<U> cast(Output<T> value, Class<U> type) {
+      DataType dtype = DataType.fromClass(type);
+      return g.opBuilder("Cast", "Cast")
+          .addInput(value)
+          .setAttr("DstT", dtype)
+          .build()
+          .<U>output(0);
+    }
+
+    Output<UInt8> decodeJpeg(Output<String> contents, long channels) {
+      return g.opBuilder("DecodeJpeg", "DecodeJpeg")
+          .addInput(contents)
+          .setAttr("channels", channels)
+          .build()
+          .<UInt8>output(0);
+    }
+
+    <T> Output<T> constant(String name, Object value, Class<T> type) {
+      try (Tensor<T> t = Tensor.<T>create(value, type)) {
+        return g.opBuilder("Const", name)
+            .setAttr("dtype", DataType.fromClass(type))
+            .setAttr("value", t)
+            .build()
+            .<T>output(0);
+      }
+    }
+    Output<String> constant(String name, byte[] value) {
+      return this.constant(name, value, String.class);
+    }
+
+    Output<Integer> constant(String name, int value) {
+      return this.constant(name, value, Integer.class);
+    }
+
+    Output<Integer> constant(String name, int[] value) {
+      return this.constant(name, value, Integer.class);
+    }
+
+    Output<Float> constant(String name, float value) {
+      return this.constant(name, value, Float.class);
+    }
+
+    private <T> Output<T> binaryOp(String type, Output<T> in1, Output<T> in2) {
+      return g.opBuilder(type, type).addInput(in1).addInput(in2).build().<T>output(0);
+    }
+
+    private <T, U, V> Output<T> binaryOp3(String type, Output<U> in1, Output<V> in2) {
+      return g.opBuilder(type, type).addInput(in1).addInput(in2).build().<T>output(0);
+    }
+    private Graph g;
+  }
+}
```

Comment:
As discussed [here](https://github.com/tensorflow/java/pull/1#issuecomment-536228429), let's remove the `tensorflow-core-examples` artifact, we'll keep our samples in `github.com/tensorflow/java-models` instead.

In addition, this sample is super outdated and shouldn't be followed as an example anymore...

Reply:
Hum, ok, I was thinking that we could leave it here until we move it somewhere else, but if we're going to delete it anyway, let's remove it I suppose?

Reply:
yes let's remove it, like I said anyway, it is now a very bad example to look at.